update_employees.py

```diff
@@ -431,9 +430,13 @@ def main():
             app.record(data=record, method=method, obj=knack_obj)
         except requests.HTTPError as e:
             if e.response.status_code == 400:
-                errors_list = e.response.json()["errors"]
-                result["errors"].append(format_errors(errors_list, record))
-                continue
+                if record["field_230"] == "Crossing Guard":
+                    logging.info(f"Error with Crossing Guard record {record[email_field]['email']}, skipped record {method}")
+                    continue
+                else:
+                    errors_list = e.response.json()["errors"]
+                    result["errors"].append(format_errors(errors_list, record))
+                    continue
```

Comment:
Another option would be to skip adding crossing guard records to the payload entirely

Reply:
skipping this edge case seems best to me. i guess we'll know when this is a problem when HR asks us why some crossing guard can't do their SSPR?

Reply:
crossing guards SSPRs??